server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -251,16 +251,16 @@ private DivTag renderStatusOptionsSelector(StatusDefinitions statusDefinitions)
     dropdownTag.with(
         option(enUsMessages.at(MessageKey.DROPDOWN_PLACEHOLDER.getKeyName()))
             .isDisabled()
-            .isSelected());
+            .withCondSelected(!application.getLatestStatus().isPresent()));
 
     // Add statuses in the order they're provided.
-    statusDefinitions
-        .getStatuses()
+    statusDefinitions.getStatuses().stream()
+        .map(StatusDefinitions.Status::statusText)
         .forEach(
-            status -> {
-              String value = status.statusText();
-              OptionTag optionTag = option(value).withValue(value);
-              dropdownTag.with(optionTag);
+            statusText -> {
+              boolean isCurrentStatus = statusText.equals(application.getLatestStatus().orElse(""));
```

Comment:
suggestion: pull `application.getLatestStatus().orElse("")` into a readable var before the stream operation